tests/create_guardduty_member/main.tf

```
@@ -21,28 +21,32 @@ resource "aws_guardduty_detector" "administrator" {
 # - Creates a GuardDuty detector for the member account
 # - Creates a GuardDuty member resource in the administrator account which imnvites the member account to join the administrator account GuardDuty organization.
 # - Creates a GuardDuty invite accepter in the member account to accept the invite from the administrator account
+
+module "guardduty" {
+  source = "../../"
+
+}
+
 module "guardduty_member" {
   source = "../../modules/member"
 
-  enable                       = true
-  enable_s3_protection         = true
-  enable_kubernetes_protection = true
-  enable_malware_protection    = true
   finding_publishing_frequency = "SIX_HOURS"
 
   providers = {
     aws               = aws
     aws.administrator = aws.administrator
   }
 
+  account_id = data.aws_caller_identity.member.account_id
```

This needs to be passed through as an output of `module.guardduty`.


```suggestion
  account_id = module.guardduty.detector.account_id
```

---

I think this actually moves under the `member` block now